src/app/(auth)/forgot-password/page.tsx

```diff
@@ -32,39 +32,44 @@ export default function ForgotPassword() {
     setEmailSentCount(emailSentCount + 1);
   };
 
-  return emailSentCount === 0 ? (
+  return (
     <SmallCardForm onSubmit={handleForgotPassword}>
-      <H1>Forgot Password</H1>
-      <TextInput
-        label="Email"
-        placeholder="email@example.com"
-        type="email"
-        id="email"
-        errorText={emailError}
-        value={email}
-        setValue={setEmail}
-      />
-      <BigBlueButton type="submit">
-        <H4 $color="white">Send link to email</H4>
-      </BigBlueButton>
-    </SmallCardForm>
-  ) : (
-    <SmallCardForm onSubmit={handleForgotPassword}>
-      <Flex $direction="column" $gap="20px">
-        <H2>A password reset link has been sent to your email.</H2>
-        <H4 $color={COLORS.greyDark}>
-          This link will direct you to the next step. If you didn’t receive an
-          email, please click Resend Email.
-        </H4>
-        <HorizontalDiv>
+      {emailSentCount === 0 ? (
+        <>
+          <H1>Forgot Password?</H1>
```

Comment:
nit: for the forgot/reset password flow (but not for login/signup), the Figma actually has as font-weight: 500. We could leave it as is, but if we want to change it: 

<H1 $fontWeight='500'>Forgot Password?</H1>

Reply:
Probably best if we stay consistent (`600px`), especially since it's a pretty similar design.